pkg/search/options.go

```diff
@@ -332,8 +332,10 @@ var (
 	ReportRequestType        = newFieldLabel("Report Request Type")
 	ReportNotificationMethod = newFieldLabel("Report Notification Method")
 
-	// Event fields.
-	EventLabel = newFieldLabel("Event Label")
+	// Notification fields.
+	NotificationArea  = newFieldLabel("Notification Area")
```

Comment:
Don't really like `Area`, it doesn't like idiomatic English to me. Maybe `Domain`?